layers/+lang/go/README.org

```diff
@@ -228,6 +228,12 @@ You have a few options to ensure you always get up to date suggestions:
   ./...= for you.
 - You can configure your task runner to run =go install ./...= command on every
   file change.
+- Set =go-install-after-save= to have spacemacs run =go install --buildmode=archive= for
+  current pacakge after you saved a file:
+
+  #+begin_src emacs-lisp
+  (go :variables go-install-after-save t)
```

Comment:
Please document all the possible values of that variable both here and in the config file.